### ISSUE TITLE
project-files function needs Emacs 27

### ISSUE THREAD

**Issue by skangas** [joaotavora/eglot#785]
`package-lint-buffer' claims that:

    2829:32: error: You should depend on (emacs "27.1") if you need `project-files'.

So I guess we should either get rid of `project-files' or bump the requirement to Emacs 27.

**Comment by joaotavora** [comment]
Maybe `package-lint-buffer`  is being overzealous here. Are you sure depending on a more recent recent version of `project.el`  woudn't be sufficient? It seems at least version 0.8.1 would provide that. In that version,  Emacs's `project.el` states:

```
;;; project.el --- Operations on the current project  -*- lexical-binding: t; -*-

;; Copyright (C) 2015-2021 Free Software Foundation, Inc.
;; Version: 0.8.1
;; Package-Requires: ((emacs "26.1") (xref "1.0.2"))
```

So maybe it's just a question of bumping the `project.el`  dependency in `eglot.el`.

**Comment by skangas** [comment]
That's true, so I'll push a change bumping the project requirement to 0.8.1.

**Comment by skangas** [comment]
Actually, `project-files` is available already from `project` version 0.1 on GNU ELPA, so there is no need to do anything here. Closing.
